torch/csrc/serialization.cpp

```diff
@@ -359,7 +359,7 @@ c10::intrusive_ptr<c10::StorageImpl> THPStorage_readFileRaw(
     data = static_cast<uint8_t*>(storage->mutable_data());
   } else {
     // NOLINTNEXTLINE(cppcoreguidelines-avoid-c-arrays,modernize-avoid-c-arrays)
-    cpu_data = std::unique_ptr<char[]>(new char[nbytes]);
```

Comment:
Nit: this is a dynamic array: yikes. Wouldn't a C10 small vector be better? Or why not just an std::string? Both are going to be heap allocated anyway.